extension/wasm_src/src/lib.rs

```diff
@@ -91,9 +87,8 @@ impl Default for DiagramOptions {
 /// Parse the given macro_rules!()-source, returns an SVG and it's preferred width
 #[wasm_bindgen(js_name = toDiagram)]
 pub fn to_diagram(src: &str, options: &DiagramOptions) -> Result<Diagram, JsValue> {
-    log(&format!("{:?}", options));
     let macro_rules = macro_railroad::parser::parse(src)
-        .map_err(|_| "Macro railroad parse failed.".to_string())?;
+        .map_err(|_| format!("macro_railroad parse failed: {}", e))?;
```

Comment:
This will fail to compile: `|_|` should `|e|` :-)

Reply:
Ooops. My bad.